internal/pkg/daemon/bpfrecorder/bpfrecorder.go

```diff
@@ -44,6 +34,15 @@ import (
 	"google.golang.org/grpc/credentials/insecure"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/kubernetes"
+	"os"
+	"runtime"
+	"sort"
+	"strconv"
+	"strings"
+	"sync"
+	"sync/atomic"
+	"syscall"
+	"time"
```

Comment:
Those should go above the non-native imports (like it was before).

Reply:
ok i have restored them to their original position.